include/fmt/format.h

```diff
@@ -391,11 +391,14 @@ class uint128_fallback {
       hi_ += (lo_ < n ? 1 : 0);
       return *this;
     }
-#if FMT_HAS_BUILTIN(__builtin_addcll)
+#if FMT_XLC_VERSION && defined(_ARCH_PWR9)
+    lo_ = __addex(lo_, n, 0, &carry);
+    hi_ += carry;
+#elif FMT_HAS_BUILTIN(__builtin_addcll) && !FMT_XLC_VERSION
     unsigned long long carry;
     lo_ = __builtin_addcll(lo_, n, 0, &carry);
     hi_ += carry;
-#elif FMT_HAS_BUILTIN(__builtin_ia32_addcarryx_u64)
+#elif FMT_HAS_BUILTIN(__builtin_ia32_addcarryx_u64) && !FMT_XLC_VERSION
```

Comment:
Does `__builtin_ia32_addcarryx_u64` need to be excluded?

Reply:
as you removed the line
```
FMT_XLC_VERSION && defined(_ARCH_PWR9)
```
and in particular `defined(_ARCH_PWR9)` you don't need ` && !FMT_XLC_VERSION` anymore

Reply:
edit: I misunderstand the issue. `__builtin_ia32_addcarryx_u64` must be skipped to avoid the same problem of `__builtin_addcll`